src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java

```diff
@@ -385,4 +395,100 @@ public String generateBearerToken(String bearer, String role) {
         return bearer;
     }
 
+    @SuppressWarnings("unchecked")
+    public String searchUserStatus(String orgIdentifier, String userId) {
```

Comment:
do we need @suppresswarning annotation?

Reply:
actually added because , intellij was complaining , but removed as its not production code